src/utils/useSpringAnimation.ts

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { useSpring } from "@react-spring/web";
+
+export function useSpringAnimation(ref: React.RefObject<HTMLDivElement>) {
```

Comment:
two things:
- unless you plan to reuse it, it should probably live next to the Card component
- it is quite specific, so maybe name it `useCardSpringAnimation`